plat/arm/board/fvp/fvp_trusted_boot.c

```diff
@@ -5,7 +5,7 @@
  */
 
 #include <assert.h>
-#include <platform_oid.h>
+#include <tbbr_oid.h>
```

Comment:
Please move the include to be in alphabetical order?
Also, please update the copyright header to 2016-2017.